tests/test-common.js

```diff
@@ -1006,6 +1008,48 @@ describe('JSON-LD Signatures', () => {
           };
           assert.deepEqual(result, expected);
         });
+        it('should sign and verify with out a controller passed to purpose',
+          async () => {
+          const Suite = suites[suiteName];
+          const signSuite = new Suite({
+            ...mock.suites[suiteName].parameters.assertionMethod,
```

Comment:
If these are supposed to be params for an assertionMethod proof ... why is the proof purpose an AuthenticationProofPurpose below? Should the params be renamed to something like `keyWithControllerObject`?

Reply:
I changed it `controllerObject` hope that is descriptive enough: https://github.com/digitalbazaar/jsonld-signatures/pull/91/commits/c7f463c7169addf6bcdb0dec6f35f64b7994a3ac